google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -457,7 +457,6 @@ private void appendLoop() {
             && !streamName.isEmpty()
             && !originalRequest.getWriteStream().equals(streamName)) {
           streamName = originalRequest.getWriteStream();
-          writerSchema = originalRequest.getProtoRows().getWriterSchema();
```

Comment:
Wondering why is this named originalRequest? Should it be currentRequest?

Reply:
The originalRequest represent the request content before modifying,
the currentRequest lacks this level of meaning. So I feel like originalRequest is more close to the meaning here.